### PR TITLE
feat: event creation screen works

### PR DESCRIPTION
**Pull request by AlbertoCentonze**
# What I did
Design of the screen has been refined, now the backend is updated with the actual values from the form.

# How I did it
- Fixed the style classes to resemble more closely the figma
- Connected the `DescriptionScreen` through navigation and access its values from the `RegistrationContext`

# How to verify it
- Try to make an event
- Run unit tests

# Video demo
https://github.com/uniconnect-epfl/uniconnect/assets/11707683/0af17a29-8c93-49d5-9c1d-9c0f50c615da

# Pre-merge checklist
The changes I have introduced:
- [x] work correctly
- [x] do not break other functionalities
- [x] work correctly on Android
- [x] are fully tested

### CODE REVIEW

**Review by Aidasvenc — APPROVED** [review]
Very clean code! Thank you!

**Review by Fyuryy — APPROVED** [review]
I just tested it using an android emulator and there are a couple issues to address. As the screen itself work I will approve the PR but here's a list of issues to address:

- While scrolling in the announcement screen, it selects the announcement where your mouse is when you stop scrolling.
- There is no "create Announcement" button
- When you click Validate, you should reset the screen and go back to the Event Screen.

File: screens/EventCreation/EventCreationScreen.tsx [hunk 1]
```diff
@@ -1,84 +1,86 @@
...
-  //  const [point, setPoint] = useState({ x: 47.238458, y: 5.984155 })
+  const { description, setDescription } = useContext(RegistrationContext)
 
+  const opacity = !hasBeenTouched ? 0.2 : 1
 
-  /* const newEvent = async () => {
-     console.log("Creating event")
-     await createEvent("uid2", title, description, new Date(2025, 0, 1), { x: 47.238458, y: 5.984155 }, location, "imageUrl")
-  }*/
+  const publish = async () => {
```
[the remark this answers is not in the document]
Comment (AlbertoCentonze) [reply screens/EventCreation/EventCreationScreen.tsx:34]: Not sure of what you are talking about.. Can you expand?

File: screens/EventCreation/EventCreationScreen.tsx [hunk 2]
```diff
@@ -1,84 +1,86 @@
...
 
   return (
-    <View style={styles.form}>
-      <View style={[styles.header, {paddingTop: insets.top + 5}]}>
-        <TouchableOpacity
-          onPress={() => navigation.goBack()}
-          testID="back-button"
-        >
+    <ScrollView style={styles.container}>
+      <View style={[styles.header, { paddingTop: insets.top + 5 }]}>
```
[the remark this answers is not in the document]
Comment (AlbertoCentonze) [reply screens/EventCreation/EventCreationScreen.tsx:72]: I just merged, this is not my code (I received it from a merge)...While I agree with you I will solve this in a future sprint to close this one.

**Pull request merged**